### ISSUE TITLE
`KeyError: 'builtins'` When Automatic configuration django-cotton

### ISSUE THREAD

**Issue by Macktireh** [wrabit/django-cotton#91]
While following the documentation for configuring Automatic django-cotton, I encountered an error during execution. The error is as follows:

```PY
File "E:\develop\projects\saas\BienRental\.venv\Lib\site-packages\django_cotton\apps.py", line 67, in ready
    wrap_loaders("django")
  File "E:\develop\projects\saas\BienRental\.venv\Lib\site-packages\django_cotton\apps.py", line 46, in wrap_loaders
    template_config["OPTIONS"]["builtins"].insert(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'builtins'
```

It appears that the issue is related to accessing the `'builtins'` key in the `template_config["OPTIONS"]` dictionary. This key does not exist, resulting in a `KeyError`.

**Comment by Macktireh** [comment]
I temporarily resolved the issue by adding `"builtins": []` to the `TEMPLATES` setting:

```py
TEMPLATES = [
    {
        "BACKEND": "django.template.backends.django.DjangoTemplates",
        "DIRS": [
            os.path.join(BASE_DIR, "templates"),
        ],
        "APP_DIRS": True,
        "OPTIONS": {
            "context_processors": [
                "django.template.context_processors.debug",
                "django.template.context_processors.request",
                "django.contrib.auth.context_processors.auth",
                "django.contrib.messages.context_processors.messages",
                "apps.common.context_processors.common_processor",
            ],
            "builtins": [],
        },
    },
]
```

**Comment by Macktireh** [comment]
It might be better to use a `try-except` block in `django_cotton/apps.py`:

```py
if not builtins_already_configured:
    try:
        template_config["OPTIONS"]["builtins"].insert(
            0, "django_cotton.templatetags.cotton"
        )
    except KeyError:
        template_config["OPTIONS"]["builtins"] = ["django_cotton.templatetags.cotton"]
```

This way, if the `'builtins'` key is missing, it will be created with the appropriate value, preventing the `KeyError`.

**Comment by wrabit** [comment]
Thanks @Macktireh, fixed in #92 and will be released soon
